docs/src/prototypes/chatPane/chatPaneHeader.tsx

```diff
@@ -87,7 +86,7 @@ class ChatPaneHeader extends React.PureComponent<IChatPaneHeaderProps> {
             styles={{
               fontWeight: 100,
               ...(!index && { marginRight: '1.6rem' }),
-              marginTop: pxToRem(8),
+              marginTop: '8px',
```

Comment:
Just out of curiosity what is the need of this? Is the result some decimal number?

Reply:
the space was not correct in initial implementation + consistency in the prototypes, the devs using them might don't have access to the pxToRem utility

Reply:
not sure if we should advertise the pixel-based values in prototypes. The question we need to ask ourselves is the following - **which units would be correct here, pixels or rems**? It seems that `rems` would be a better answer, given that
-  all Stardust components scales in accordance to `rem` units
- thus, client's intent here would really to introduce `rem`-based value here, so this indent would scale with the component being scaled
- as a result, I would expect that if not the `pxToRem`, but at least `rem`-based value will be specified here

Reply:
fixed by providing margin auto